### ISSUE TITLE
Email content rendered always as text

### ISSUE THREAD

**Issue by pablocrivella** [markets/maily#32]
Suddenly emails content is only rendered as text. Any idea why this could be?

<img width="1382" alt="maily - account mailer email changed 2018-12-19 07-34-05" src="https://user-images.githubusercontent.com/1854923/50217919-94d15000-0360-11e9-8103-cfdd098d5e7b.png">

Also, is there anyway to update the emails without having to restart the rails server?

TIA!

**Comment by markets** [comment]
@pablocrivella I think I've found the cause: `mail` gem updated from 2.7.0 to 2.7.1 (released latest month), returning back to previous version fixed it for me:

```ruby
gem 'mail', '2.7.0'
```

You can use this workaround for now, but I'll try to find a fix :)

This is what changed:

- In 2.7.0:
```
content.raw_source.html_safe? => true
```

- In 2.7.1:
```
content.raw_source.html_safe? => false
```

**Comment by markets** [comment]
@pablocrivella I just pushed a fix here #33 :stuck_out_tongue: could you please give it a try?

**Comment by markets** [comment]
@pablocrivella v0.8.2 released https://rubygems.org/gems/maily/versions/0.8.2 🚀
